.vuepress/public/_redirects

```diff
@@ -80,5 +80,17 @@
 /reference/features/filtering.html                   /reference/features/filtering_and_faceted_search.html
 /reference/features/faceted_search.html              /reference/features/filtering_and_faceted_search.html
 
-#move updating MeiliSearch
+# Move under the hood articles to /learn
+/reference/under_the_hood/concat.html                /learn/advanced/concat.html
+/reference/under_the_hood/datatypes.html             /learn/advanced/datatypes.html
+/reference/under_the_hood/indexation.html            /learn/advanced/indexation.html
+/reference/under_the_hood/prefix.html                /learn/advanced/prefix.html
+/reference/under_the_hood/storage.html               /learn/advanced/storage.html
+/reference/under_the_hood/tokenization.html          /learn/advanced/tokenization.html
+/reference/under_the_hood/typotolerance.html         /learn/advanced/typotolerance.html
+
+# Bucket sort
+/reference/under_the_hood/bucket_sort                /learn/core_concepts/relevancy.html#behavior
```

Comment:
Missed one 😉 

```suggestion
/reference/under_the_hood/bucket_sort.html                /learn/core_concepts/relevancy.html#behavior
```

Reply:
Thanks for rebasing btw ❤️

Reply:
We had a couple of conflicts due to the latest PR moving the Update guide

Reply:
Also, thanks for catching that missing `.html`